src/utils/datastructs.jl

```diff
@@ -7,13 +7,13 @@ abstract type ClustResult end
 "FullInputData"
 struct FullInputData <: TSData
  region::String
- years::Array{Int64}
+ years::Array{Int,1}
  N::Int
  data::Dict{String,Array}
 end
 
 """
-      ClustData{region::String,K::Int,T::Int,data::Dict{String,Array},weights::Array{Float64},mean::Dict{String,Array},sdv::Dict{String,Array}} <: TSData
+      ClustData{region::String,K::Int,T::Int,data::Dict{String,Array},weights::Array{Float64,2},mean::Dict{String,Array},sdv::Dict{String,Array},delta_t::Array{Float64,2},k_ids::Array{Int}} <: TSData
```

Comment:
I fear we forgot to remove the ,2 here again. Shall I change it in the master or in the dev later.